fe/fe-core/src/main/java/com/starrocks/connector/iceberg/io/IcebergCachingFileIO.java

```diff
@@ -236,7 +314,14 @@ private SeekableInputStream cachedStream() throws IOException {
             try {
                 CacheEntry entry = contentCache.get(location(), k -> newCacheEntry());
```

Comment:
This cache key looks like diff from loadMetadataDiskCache.  the cache key has s3 prefix here?

Reply:
cache key is  s3a  or  some format  request.    load  from local disk path to cache , key will transfer to s3a

Reply:
So if location() return a hdfs path, this will not hit the cache?